config/read.go

```diff
@@ -221,6 +221,9 @@ func getDefaultConfig() *ServerConfig {
 	if gcpProjectID := os.Getenv("GCP_PROJECT_ID"); gcpProjectID != "" {
 		config.GcpProjectID = gcpProjectID
 	}
+	if crunchyBridgeAPIKey := os.Getenv("CRUNCHY_BRIDGE_API_KEY"); crunchyBridgeAPIKey != "" {
+		config.CrunchyBridgeAPIKey = crunchyBridgeAPIKey
+	}
```

Comment:
We should probably also add this for setting `crunchy_bridge_cluster_id` through an environment variable, looks like I missed that at the time.